sources/packages/backend/apps/api/src/route-controllers/assessment/assessment.controller.service.ts

```diff
@@ -207,7 +214,7 @@ export class AssessmentControllerService {
     );
     const [firstDisbursement, secondDisbursement] =
       assessment.disbursementSchedules;
-    let finalAward = {};
+    let finalAward: Record<string, string | number> = undefined;
```

Comment:
I believe, we no need to mention = undefined, as the variable is explicitly defined as Record<string, string | number>

Reply:
I may be missing something but I did not follow the point here. The `undefined` is set to make it explicit that it will be returned as `undefined`, for instance, if the next `if` condition is not satisfied.
Can you please clarify why we would not declare the type in this situation?

Reply:
i got your point.